projects/netgrif-components/schematics/view/create-view-prompt/views/public-task-view/create-public-task-view.ts

```diff
@@ -0,0 +1,46 @@
+import {chain, Rule, Tree} from '@angular-devkit/schematics';
+import {strings} from '@angular-devkit/core';
+import {createFilesFromTemplates, createRelativePath, getProjectInfo} from '../../../../_utility/utility-functions';
+import {updateAppModule} from '../../../_utility/view-utility-functions';
+import {addViewToViewService} from '../../../_utility/view-service-functions';
+import {TabbedView} from '../../models/tabbed-view';
+import {ViewClassInfo} from '../../../../_commons/view-class-info';
+import {ImportToAdd} from '../../../../_commons/import-to-add';
+import {CreateTaskViewArguments} from '../../models/create-task-view-arguments';
+
+
+export function createPublicTaskView(tree: Tree, args: CreateTaskViewArguments & TabbedView, addViewToService: boolean): Rule {
+    const projectInfo = getProjectInfo(tree);
+    const view = new ViewClassInfo(
+        args.path,
+        args.viewType,
+        args.componentName);
+    const rules = [];
+
+    const templateParams = {
```

Comment:
unused template parameters

Reply:
we use simillar template params in every create view schematic, these ones was copied from task view schematic, it should be removed ? then we should remove it on all create view schematics

Reply:
done